src/components/unique-token/UniqueTokenAttributeItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { UniqueAssetTrait } from '../../entities/uniqueAssets';
+import Tag from '../Tag';
+import transformUniqueAssetTraitsForPresentation from '@rainbow-me/helpers/transformUniqueAssetTraitsForPresentation';
+import { magicMemo } from '@rainbow-me/utils';
+
+type AttributeItemProps = UniqueAssetTrait &
+  ReturnType<typeof transformUniqueAssetTraitsForPresentation> & {
+    color: string;
+    slug: string;
+    max_value?: string | number;
+  };
+
+const UniqueTokenAttributeItem: React.FC<AttributeItemProps> = ({
```

Comment:
mmm, what's the purpose of this wrapper? Can't we just use `Tag` instead in `src/components/unique-token/UniqueTokenAttributes.tsx`?

Reply:
okay, I see, this is making this mapping. But I guess we can dot his directly in `Tag` no?

Reply:
I think we could skip it in some way @terrysahaidak what do you think about it since right now it lost the logic it had before?

Reply:
Actually whatever Terry says I think it makes sense to leave that. Tag is a UI component and uses it's own naming that is very close to the visual part and is coherent with that. I think this mapping is actually not something huge and it is okay to have it to make Tah code more independent from OpenSea API. Let me know what you think

Reply:
what?

Reply:
We don't use tags in any other context. This is a component made purely for OpenSea API. If you want to do mapping anyway, it's ok, but why then not do it at the parent component level?